internal/build/imgsrc/buildkit_builder.go

```diff
@@ -76,18 +87,27 @@ func (r *BuildkitBuilder) buildWithBuildkit(ctx context.Context, streams *iostre
 		span.End()
 	}()
 
+	buildkitAddr := r.addr
+	if buildkitAddr == "" {
+		_, app, err := r.provisioner.EnsureBuilder(ctx, "", false)
```

Comment:
we need to properly pass region and recreateBuilder args through in order for the associated flags to work properly (eg `--recreate-builder`)

Reply:
I'm gonna pass the parameters explicitly from buildkit_builder.go for now. Passing them to the Provisioner makes sense though.